### PR TITLE
[WIP] ROC-2471: 'Make an offer' landing page

### PR DESCRIPTION
**Pull request by rlewan**
### JIRA link ###

https://tools.hmcts.net/jira/browse/ROC-2471

### Change description ###

Landing page for 'Make an offer' feature and links to it.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/features/response/views/confirmation.njk [hunk 1]
```diff
@@ -8,51 +8,74 @@
...
 
       <div>
         <h2 class="heading-medium">{{ t("What happens next") }}</h2>
         <p>
-          {{ t("The case will be reviewed by a judge and might go to a hearing. We’ll contact you if a hearing date is scheduled and explain what you need to do to prepare.") }}
+          {{ t(
+            '%s has until %s to respond to your defence.',
+            {
+              postProcess: 'sprintf',
+              sprintf: [claimantName, '!!! TODO Calculate this date !!!']
```
Comment (timja-kainos) [inline src/main/features/response/views/confirmation.njk]: Calculate this date?

File: src/main/features/response/views/confirmation.njk [hunk 2]
```diff
@@ -8,51 +8,74 @@
...
           ) }}
         </p>
-
-        {{ supportWidget() }}
+        <p>
+          {{ t(
+            'You can still contact %s to make a <a href="%s">settlement out of court</a>. If you agree a settlement then the claim is closed and you won\'t have to go to a hearing.',
+            {
+              postProcess: 'sprintf',
+              sprintf: [claimantName, '/offer/todo/add/the/page']
```
Comment (timja-kainos) [inline src/main/features/response/views/confirmation.njk]: can you add the page?

File: src/main/features/response/views/confirmation.njk (hunk 1 above)
Comment (rlewan) [reply src/main/features/response/views/confirmation.njk]: I'm waiting for Art to respond on this. We don't currently have a config for number of days claimant has to respond to a defence.

File: src/main/features/response/views/confirmation.njk (hunk 2 above)
Comment (rlewan) [reply src/main/features/response/views/confirmation.njk]: Maybe ;)

**Comment by rlewan** [comment]
Scrapping this as [ROC-2471](https://tools.hmcts.net/jira/browse/ROC-2471) is a duplicate of [ROC-2244](https://tools.hmcts.net/jira/browse/ROC-2244).

**Pull request closed without merging**